Formula/p/paperjam.rb

```diff
@@ -0,0 +1,26 @@
+class Paperjam < Formula
+  desc "Program for transforming PDF files"
+  homepage "https://mj.ucw.cz/sw/paperjam/"
+  url "https://mj.ucw.cz/download/linux/paperjam-1.2.tar.gz"
+  sha256 "d00ce24fb44e2024c43ca680e146f507c36d21f2e6c515eb896edca11c43a50d"
+  license "GPL-2.0-or-later"
+
+  depends_on "asciidoc" => :build
+  depends_on "docbook-xsl" => :build
+  depends_on "libpaper"
+  depends_on "qpdf"
+
+  fails_with gcc: "5" # C++17
```

Comment:
It would need GCC 11 or later, but the output suggests it's using that.
```console
==> make
g++-11 -O2 -Wall -Wextra -Wno-parentheses -std=gnu++11 -g -DVERSION='"1.2"' -DYEAR='"2022"' -DBUILD_DATE='"2024-08-20"' -DBUILD_COMMIT='"<unknown>"'   -c -o paperjam.o paperjam.cc
g++-11 -O2 -Wall -Wextra -Wno-parentheses -std=gnu++11 -g   -c -o pdf-tools.o pdf-tools.cc
g++-11 -O2 -Wall -Wextra -Wno-parentheses -std=gnu++11 -g   -c -o parse.o parse.cc
g++-11 -O2 -Wall -Wextra -Wno-parentheses -std=gnu++11 -g   -c -o cmds.o cmds.cc
```

Reply:
Ah, the makefile hardcodes this:
```sh
CXXFLAGS=-O2 -Wall -Wextra -Wno-parentheses -std=gnu++11 -g
```